mergebot-config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "label": {
+    "enabled": true,
+    "interactive": false,
+    "default": "requested-changes"
```

Comment:
This shouldn't be `requested-changes` see here: https://github.com/joerg84/universe/pull/2